src/main/java/frc/robot/Robot.java

```diff
@@ -32,15 +32,27 @@ public class Robot extends LoggedRobot {
   public static final boolean isReplayMode = false;
   public static final double kDefaultPeriod = 0.02;
 
+  public static final int ledPort = 2;
+  public static final int ledStripLength = 256;
+  public static final int ledMatrixLenth = 256;
+
   //RoboRIO serial numbers
   public static final String competitionRobotSerial = "03161743";;
   public static final String practiceRobotSerial = "03064df0";
   public static final boolean isCompetitionRobot = !HALUtil.getSerialNumber().equals(practiceRobotSerial);
 
+  public static enum LedEnum {
+    BLINKIN, STRIP, MATRIX, NONE;
+  }
+
   //Subsystem toggle
+  public static final LedEnum ledSubSelect = LedEnum.STRIP;
```

Comment:
Should this be 3 separate booleans for the 3 separate subsystems? Or are we going to use this enum

Reply:
The subsystems are mutually exclusive. Only one can actually exist because only one LED driver is allowed on the roboRIO (not really sure why). The enum enforces this exclusivity better than three booleans.

Reply:
Got it, didn't know they were mutually exclusive